external-crates/move/crates/move-cli/src/base/stackless.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) The Move Contributors
```

Comment:
I am not the largest fan of exposing this as a top-level tool -- do @tzakian  or @tnowacki have thoughts?